themes/default/content/blog/automating-pulumi-import-to-bring-manually-created-resources-into-iac/index.md

```diff
@@ -0,0 +1,196 @@
+---
+title: "Automating Pulumi Import with Manually Created Resources"
```

Comment:
Something about the way this is phrased makes it sound like "manually created resources" are the feature -- like we're using manually created resources to automate Pulumi import, which doesn't quite capture it. I don't have a great alternative at the ready, but will think on it -- just wanted to give that feedback, though, since you asked specifically for it.

Reply:
I'm in the same boat - I am pretty happy with the content but really struggling for a fitting title.

Reply:
Gonna mark as resolved for now since neither of us has something clearly better.